### ISSUE TITLE
Alternate way of dealing with dates in /data-exploratory-analysis.html section 3.1

### ISSUE THREAD

**Issue by lukestein** [aeturrell/coding-for-economists#11]
As someone somewhat experienced with EDA and data cleaning but new-ish to this work in python, I was interested in learning your (perhaps pythonic) solution:
```python
start_code = 16436
end_code = df['Date'].max() + 1 # +1 because of how ranges are computed; we want to *include* the last date

datetime_dict = dict(zip(range(start_code, end_code),
                               pd.date_range(start='2005/01/01', periods=end_code-start_code)))

df['datetime'] = df['Date'].apply(lambda x: datetime_dict[x])
```

but thought I'd mention that the solution that occurred to me first and seems perhaps easier both to develop and explain was
```python
def convert_date(d):
    return pd.to_datetime("01-01-2005") + pd.DateOffset(d-16436)

df['datetime'] = df['Date'].apply(convert_date)
```

**Comment by aeturrell** [comment]
No, yours is better! This should replace the original.

**Comment by aeturrell** [comment]
Fixed in da85610e30418fb69cf874d2e2bc227eb5730c6b
